heka/sandbox/filters/firefox_aggregator.lua

```diff
@@ -0,0 +1,193 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Collects the cbufd, txt, and json output from multiple instances of an upstream
+Firefox sandbox filter (the filters should all be the same version at least with
+respect to their cbuf output). The purpose is to recreate the view at a larger
+scope in each level of the aggregation
+i.e., host view -> datacenter view -> service level view.
+
```

Comment:
txt and json will just duplicate the docs in agg.lua module and cbuf aggregation will just duplicate the circular buffer docs.